docs/source/installation-guide.rst

```diff
@@ -73,6 +73,15 @@ directory like this:
 	cd ${QUICK_HOME}
 	mkdir builddir
 
+check CMake version
+^^^^^^^^^^^^^^^^^^^^^^
+Ensure CMake (version 3.12.0 or higher) is installed::
+
+	cmake --version
+
+If CMake is not installed on macOS, you can install it using Homebrew::
```

Comment:
Since this is macOS specific, we should remove it from here. Either delete or create a new section like "macOS tips".